container-search/src/main/resources/configdefinitions/qr-start.def

```diff
@@ -9,7 +9,7 @@ jvm.server bool default=true restart
 jvm.verbosegc bool default=true restart
 
 ## Garbage Collection tuning parameters
-jvm.gcopts string default="" restart
+jvm.gcopts string default="-XX:+UseConcMarkSweepGC -XX:MaxTenuringThreshold=15 -XX:NewRatio=1" restart
```

Comment:
Consider updating `vespa-start-container-daemon.sh` to match this default.

Reply:
Done